frontend/src/scenes/error-tracking/errorTrackingDataNodeLogic.tsx

```diff
@@ -22,48 +22,48 @@ export const errorTrackingDataNodeLogic = kea<errorTrackingDataNodeLogicType>([
     })),
 
     actions({
-        mergeGroups: (indexes: number[]) => ({ indexes }),
-        assignGroup: (recordIndex: number, assigneeId: number | null) => ({
+        mergeIssues: (indexes: number[]) => ({ indexes }),
```

Comment:
I kind of wish these were issue id's, rather than indexes in a table, just because this would feel more reusable then - but I've no idea how feasible that is.

Reply:
I did want to use ids not issues but I remember when writing this that it required some nasty looping to find the records or something. Can't fully remember the context. Perhaps that's easier now given we're comparing ids and not fingerprints (which were arrays of strings at the time)

Reply:
Actually just went ahead and fixed it

Reply:
Yep, not blocking for sure!